nav2_dijkstra_planner/README.md

```diff
@@ -0,0 +1,25 @@
+# Dijkstra Planner
+The DijkstraPlanner is an implementation of a [Planning module](../doc/requirements/requirements.md) (equivalent to GlobalPlanner in ROS1 [MoveBase](http://wiki.ros.org/move_base)).
```

Comment:
I'd say that the DijkstraPlanner is a planning module that implements the ComputePathToPose interface.

Reply:
Thanks, I'll make the change.